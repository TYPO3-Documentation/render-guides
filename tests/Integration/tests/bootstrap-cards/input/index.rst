==============
Simple cards
==============

..  container:: row m-0 p-0

    ..  container:: col-md-6 pl-0 pr-3 py-3 m-0

        ..  container:: card px-0 h-100

            ..  rst-class:: card-header h3

                ..  rubric:: **Concepts**

            ..  container:: card-body

                Written for new users, this chapter introduces some of TYPO3s
                core concepts including the backend, TYPO3s administration
                interface.

    ..  container:: col-md-6 pl-0 pr-3 py-3 m-0

        ..  container:: card px-0 h-100

            ..  rst-class:: card-header h3

                ..  rubric:: **System Requirements**

            ..  container:: card-body

                System requirements for the host operation system, including
                its web server and database and how they should be configured
                prior to installation.

Cards with buttons in the footer
================================

..  container:: row m-0 p-0

    ..  container:: col-md-6 pl-0 pr-3 py-3 m-0

        ..  container:: card px-0 h-100

            ..  rst-class:: card-header h3

                ..  rubric:: **Pages**

            ..  container:: card-body

                The Page Management Guide introduces TYPO3's Page Tree and explains how pages are created and managed.

            ..  container:: card-footer pb-0

                ..  rst-class:: horizbuttons-striking-m

                *   `12-dev <https://docs.typo3.org/m/typo3/tutorial-editors/main/en-us/Pages/Index.html>`__
                *   `11.5 <https://docs.typo3.org/m/typo3/tutorial-editors/11.5/en-us/Pages/Index.html>`__
                *   `10.4 <https://docs.typo3.org/m/typo3/tutorial-editors/10.4/en-us/Pages/Index.html>`__

    ..  container:: col-md-6 pl-0 pr-3 py-3 m-0

        ..  container:: card px-0 h-100

            ..  rst-class:: card-header h3

                ..  rubric:: **Content**

            ..  container:: card-body

                The Content Creation Guide shows how page content is created in the form of Content Elements.

            ..  container:: card-footer pb-0

                ..  rst-class:: horizbuttons-striking-m

                *   `12-dev <https://docs.typo3.org/m/typo3/tutorial-editors/main/en-us/ContentElements/Index.html>`__
                *   `11.5 <https://docs.typo3.org/m/typo3/tutorial-editors/11.5/en-us/ContentElements/Index.html>`__
                *   `10.4 <https://docs.typo3.org/m/typo3/tutorial-editors/10.4/en-us/ContentElements/Index.html>`__

