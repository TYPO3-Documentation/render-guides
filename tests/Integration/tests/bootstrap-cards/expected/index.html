<!-- content start -->
                <section class="section" id="simple-cards">
            <h1>Simple cards<a class="headerlink" href="#simple-cards" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">&nbsp;<i class="fa-solid fa-link"></i></a></h1>
            <div class="row m-0 p-0">
    <div class="col-md-6 pl-0 pr-3 py-3 m-0">
    <div class="card px-0 h-100">
    <div class="rubric card-header h3"><strong>Concepts</strong></div><div class="card-body">

    <p>Written for new users, this chapter introduces some of TYPO3s
core concepts including the backend, TYPO3s administration
interface.</p>

</div>

</div>

</div>
<div class="col-md-6 pl-0 pr-3 py-3 m-0">
    <div class="card px-0 h-100">
    <div class="rubric card-header h3"><strong>System Requirements</strong></div><div class="card-body">

    <p>System requirements for the host operation system, including
its web server and database and how they should be configured
prior to installation.</p>

</div>

</div>

</div>

</div>

            <section class="section" id="cards-with-buttons-in-the-footer">
            <h2>Cards with buttons in the footer<a class="headerlink" href="#cards-with-buttons-in-the-footer" data-bs-toggle="modal" data-bs-target="#linkReferenceModal"  title="Reference this headline">&nbsp;<i class="fa-solid fa-link"></i></a></h2>
            <div class="row m-0 p-0">
    <div class="col-md-6 pl-0 pr-3 py-3 m-0">
    <div class="card px-0 h-100">
    <div class="rubric card-header h3"><strong>Pages</strong></div><div class="card-body">

    <p>The Page Management Guide introduces TYPO3&#039;s Page Tree and explains how pages are created and managed.</p>

</div>
<div class="card-footer pb-0">


<ul class="horizbuttons-striking-m">
    <li><a href="https://docs.typo3.org/m/typo3/tutorial-editors/main/en-us/Pages/Index.html">12-dev</a></li>
    <li><a href="https://docs.typo3.org/m/typo3/tutorial-editors/11.5/en-us/Pages/Index.html">11.5</a></li>
    <li><a href="https://docs.typo3.org/m/typo3/tutorial-editors/10.4/en-us/Pages/Index.html">10.4</a></li>
</ul>

</div>

</div>

</div>
<div class="col-md-6 pl-0 pr-3 py-3 m-0">
    <div class="card px-0 h-100">
    <div class="rubric card-header h3"><strong>Content</strong></div><div class="card-body">

    <p>The Content Creation Guide shows how page content is created in the form of Content Elements.</p>

</div>
<div class="card-footer pb-0">


<ul class="horizbuttons-striking-m">
    <li><a href="https://docs.typo3.org/m/typo3/tutorial-editors/main/en-us/ContentElements/Index.html">12-dev</a></li>
    <li><a href="https://docs.typo3.org/m/typo3/tutorial-editors/11.5/en-us/ContentElements/Index.html">11.5</a></li>
    <li><a href="https://docs.typo3.org/m/typo3/tutorial-editors/10.4/en-us/ContentElements/Index.html">10.4</a></li>
</ul>

</div>

</div>

</div>

</div>

    </section>
    </section>
        <!-- content end -->
